tests/component/_task_modules/channels/test_ai_channels.py

```diff
@@ -0,0 +1,722 @@
+import pytest
+
+from nidaqmx import Task
+from nidaqmx._task_modules.channels.ai_channel import AIChannel
+from nidaqmx.constants import (
+    AccelChargeSensitivityUnits,
+    AccelSensitivityUnits,
+    AccelUnits,
+    ACExcitWireMode,
+    BridgeConfiguration,
+    BridgeUnits,
+    ChargeUnits,
+    CJCSource,
+    CurrentShuntResistorLocation,
+    EddyCurrentProxProbeSensitivityUnits,
+    ExcitationSource,
+    ForceIEPESensorSensitivityUnits,
+    ForceUnits,
+    LVDTSensitivityUnits,
+    ResistanceConfiguration,
+    RTDType,
+    StrainGageBridgeType,
+    StrainGageRosetteMeasurementType,
+    StrainGageRosetteType,
+    TemperatureUnits,
+    TerminalConfiguration,
+    ThermocoupleType,
+    UsageTypeAI,
+    VoltageUnits,
+)
+
+
+# Note: Tests for other channel types will be less complete given that the underlying Python
+# implementation is code-generated and the underlying C API implementation is well unit-tested.
+@pytest.mark.parametrize(
+    "desired_term_config, expected_term_cfg",
+    [
+        (TerminalConfiguration.DEFAULT, TerminalConfiguration.DIFF),
+        (TerminalConfiguration.DIFF, TerminalConfiguration.DIFF),
+        (TerminalConfiguration.RSE, TerminalConfiguration.RSE),
+    ],
+)
+@pytest.mark.parametrize("min_val, max_val", [(-10, 10), (-5, 5)])
+@pytest.mark.parametrize(
+    "units, custom_scale_name",
+    [(VoltageUnits.VOLTS, ""), (VoltageUnits.FROM_CUSTOM_SCALE, "no_scaling_scale")],
+)
+def test___ai_channel_collection___add_ai_voltage_chan___sets_channel_attributes(
+    task,
```

Comment:
Please consider adding type annotations, at least for parameters like `task` and `sim_6363_device` where we benefit from having auto-complete work correctly.

You could also type hint `chan: AIChannel = task.ai_channels.add...` if that is helpful, but that would not be necessary if the `nidaqmx` module itself had type hints.